src/www/app/environment.ts

```diff
@@ -16,23 +16,6 @@ export interface EnvironmentVariables {
   APP_VERSION: string;
   APP_BUILD_NUMBER: string;
   SENTRY_DSN: string;
```

Comment:
Nit: I think this is string|undefined now

Reply:
Done. Good catch.